telemetry/definitions/commonDefinitions.json

```diff
@@ -1287,6 +1292,11 @@
             "type": "int",
             "description": "Number of findings discovered after executing IAM Policy Checks"
         },
+        {
+            "name": "fixGenerated",
+            "type": "boolean",
+            "description": "Whether a code fix request was able to generate a fix"
```

Comment:
How is this different from `includesFix` ? https://github.com/aws/aws-toolkit-common/commit/24ebf0638f09fc9ac1a0d7343ef8d632bbcde28c

Reply:
Good catch, I will reuse this one